UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -59,6 +59,23 @@ private void OnDisable()
 		SceneManager.activeSceneChanged -= OnRoundRestart;
 	}
 
+	/// <summary>
+	/// Checks for any Lifeforms (dead or alive) that might be aboard the shuttle
+	/// </summary>
+	/// <returns>true if a lifeform is found, false if none is found</returns>
+	bool LifeformChecker()
```

Comment:
Good rule is to call methods with a verb. In this case for example: `CheckLifeforms`

Reply:
Also we specify access modifiers. At least doing it lately. So please add `private`